templates/feature.html

```diff
@@ -82,44 +82,30 @@
           {% else %}
             Enabled by default
           {% endif %}
-        {% endif %} in <b>desktop Chrome {{ feature.meta.milestone_str }}</b>
+        {% endif %}
+        {% if feature.bug_url %}
+        (<a href="{{ feature.bug_url }}">launch bug</a>)
+        {% endif %}
+        in: </br>
```

Comment:
I think in HTML empty elements CANNOT have end tags so `<br/>` is invalid.  It's the required XHTML of course though.  Anyway just write `<br>`.